### DISCUSSION TITLE
Icons text not showed

### DISCUSSION THREAD

**Discussion by karaldo** [Open-Shell/Open-Shell-Menu#1735] [Q&A] [answered]
Hello, first I must say that Open Shell works excellent for me. But in Classic Explorer the text of icons does not display, Not under nor at the side. Could this be solved?

Thanks in advance,

**Comment by ge0rdi** [comment, marked as answer]
Do you mean texts for icons in Classic Explorer toolbar?

Those texts are displayed according to `Label` fields of toolbar items. By default there are no labels so no text is displayed.

You can add/edit labels in `Classic Explorer Settings` -> `Toolbar buttons`.
Double-click on item and fill `Label` and it should be displayed in the toolbar.

**Reply by karaldo** [reply]
Hello ge0rdi. Solved! On the other hand I feel like a fool because I had the solution in front of my own nose.

Thank you very much
